test/external/external_test.go

```diff
@@ -220,17 +241,37 @@ var _ = Describe("External tests", func() {
 			Eventually(test.CommandSession.Out, time.Minute).Should(Say("Writing 2 images...\n"))
 		})
 
+		define.Then(`^the command says local cache saved 0 bytes in downloads$`, func() {
+			Eventually(test.CommandSession.Out, time.Minute).Should(Say("Local cache saved 0 out of 767322 bytes in layer downloads\n"))
```

Comment:
I don't like that our tests depend on log output, couldn't we read from the struct that we are storing?

Reply:
All this external tests do test depending on output. All of them.